### PR TITLE
Read wishlist item

### PR DESCRIPTION
**Pull request by billykern**

### CODE REVIEW

**Comment by biegel** [comment]
this LGTM 👍

**Review by biegel — APPROVED** [review]

File: tests/test_routes.py [hunk 1]
```diff
@@ -94,6 +82,39 @@ def test_create_wishlist_no_data(self):
...
+    def test_create_wishlist_no_content_type(self):
         resp = self.app.post(BASE_URL)
         self.assertEqual(resp.status_code, status.HTTP_415_UNSUPPORTED_MEDIA_TYPE)
+
+    def test_read_wishlist_item_success(self):
+        wishlist = WishlistFactory()
+        wishlist.create()
+        item = ItemFactory(wishlist_id=wishlist.id)
+        # TODO: remove prints
+        print(item.wishlist_id)
```
Comment (ZiyiHuang96) [inline tests/test_routes.py]: TODO: remove prints LOL @billykern

File: tests/test_routes.py [hunk 2]
```diff
@@ -94,6 +82,39 @@ def test_create_wishlist_no_data(self):
...
-    def test_create_pet_no_content_type(self):
+    def test_create_wishlist_no_content_type(self):
         resp = self.app.post(BASE_URL)
         self.assertEqual(resp.status_code, status.HTTP_415_UNSUPPORTED_MEDIA_TYPE)
+
+    def test_read_wishlist_item_success(self):
+        wishlist = WishlistFactory()
+        wishlist.create()
+        item = ItemFactory(wishlist_id=wishlist.id)
+        # TODO: remove prints
```
Comment (ZiyiHuang96) [inline tests/test_routes.py]: Same here @billykern

**Review by ZiyiHuang96 — CHANGES_REQUESTED** [review]
Looks perfect just some lines to be removed.

File: tests/test_routes.py (hunk 1 above)
Comment (billykern) [reply tests/test_routes.py]: thank you!

**Review by ZiyiHuang96 — APPROVED** [review]
Looks good! Ready to merge! @billykern

**Pull request merged**